modules/display.c

```diff
@@ -365,6 +367,37 @@ static gboolean backlight_ioctl(int value)
 	return status;
 }
 
+/* Set DPMS mode for X11 display to on or off
+ *
+ * @params dpms_on To force the mode into on or off (blanked) mode.
+ */
+static void display_dpms(gboolean dpms_on)
+{
+	Display* dpy = NULL;
+
+	dpy = XOpenDisplay(NULL);
+	if (dpy == NULL) {
+		dpy = XOpenDisplay(":0.0");
```

Comment:
It's better to avoid hardcoding ":0.0" here.
If we rely on Xorg, we should use NULL here and set the environment DISPLAY=... in mce startup script.

Reply:
No, mce currently starts before X. This is the graceful fallback -- honour DISPLAY, and if it's not set, fall back to `:0.0`.

Reply:
Well, you did it above.
So, what I mean is to remove this code:
```
	if (dpy == NULL) {
		dpy = XOpenDisplay(":0.0");
```
... and just add DISPLAY=:0.0 to mce init script.

Reply:
I understand what you mean, and I think this is not the right place to do it. It means that if someone starts MCE by hand, their environment will be different, and mce will behave differently.

Reply:
Now, mce currently respects the environment, and if it doesn't have DISPLAY, it will try the maemo-default X server display.